docs/02-for-app-authors/03-metainfo-guidelines/01-quality-guidelines.md

```diff
@@ -46,6 +46,10 @@ custom icon). If it's made clear that it's a third-party client it can be
 ok to mention the original service (e.g. `Tally for Plausible`), but in
 most cases, it should be avoided.
 
+### Runtime not end-of-life
+
+Your app must target a runtime that is not end-of-life (EOL). This means you should not use deprecated or obsolete runtimes, as they may not receive security updates or support.
```

Comment:
"See https://docs.flathub.org/docs/for-app-authors/runtimes for runtime support periods and expected eol dates"